CHANGES.md

```diff
@@ -24,6 +24,9 @@
   colorBarName, colorBarNorm,  colorBarMin, colorBarMax, tileLevelMin, tileLevelMax.
   (#935, #940) 
 
+* xcube serve now allows to configure custom color maps via the configuration file.
+  It supports continuous, stepwise and categorical colormaps. (#1055)
```

Comment:
Please include my specification from the PR description so that readers know how to apply the feature.

Reply:
Oops, just saw that you enhanced the docs, great! Therefore please link to it here.

Reply:
Done